.github/workflows/main.yml

```diff
@@ -0,0 +1,80 @@
+name: ci
+
+on:
+  push:
+    branches:
+    - "*"  # run for branches
+    tags:
+    - "*"  # run for tags
+  pull_request:
+    branches:
+    - "*"  # run for branches
+    tags:
+    - "*"  # run for tags
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      GROUP: weaveworksdemos 
+      COMMIT: ${{ github.sha }}
+      REPO: queue-master
+    steps:
+    - uses: actions/checkout@v2
+
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '8' # The JDK version to make available on the path.
+        java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+        architecture: x64 # (x64 or x86) - defaults to x64
+
+ 
+    #
+    #
+    # Build
+    - run: ./scripts/build.sh
+
+    #
+    #
+    # Unit test
+    - run: ./test/test.sh unit.py
+
+    #
+    #
+    # Component test
+    - run: ./test/test.sh component.py
+
+    #
+    #
+    # Upload coverage results to coveralls
+    - name: Submit Coveralls
+      env:
+        COVERALLS_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+      #run: ./test/test.sh coveralls.py
+      run: docker run --rm -v ${HOME}/.m2:/root/.m2 -v ${PWD}/test/..:/usr/src/mymaven -w /usr/src/mymaven maven:3.2-jdk-8 mvn -DrepoToken=${COVERALLS_TOKEN} -DserviceJobId=${GITHUB_RUN_ID} -Dbranch=${GITHUB_REF} -DpullRequest=${GITHUB_HEAD_REF} -DserviceName=GITHUB verify jacoco:report coveralls:report
```

Comment:
I don't remember, but is COVERALLS_TOKEN set at the organization level or project one? If it is a latter, it has to be added before merging.

Reply:
Im not sure - I will check with @cewood

Reply:
A token has now been added for this repo